index.bs

```diff
@@ -1526,39 +1525,40 @@ In order to report a {{Gamepad/mapping}} of {{GamepadMappingType/xr-standard}} t
   <thead>
     <tr>
       <th>Axes</th>
-      <th><code>xr-standard</code> Location</th>
+      <th><code>xr-standard</code> Binding</th>
       <th>Required</th>
     </tr>
   </thead>
   <tbody>
     <tr>
       <td>axes[0]</td>
-      <td>Primary Touchpad/Joystick X</td>
-      <td>Yes</td>
+      <td>Touchpad X</td>
+      <td>If thumbstick is not present</td>
     </tr>
     <tr>
       <td>axes[1]</td>
-      <td>Primary Touchpad/Joystick Y</td>
-      <td>Yes</td>
+      <td>Touchpad Y</td>
+      <td>If thumbstick is not present</td>
     </tr>
     <tr>
       <td>axes[2]</td>
-      <td>Secondary Touchpad/Joystick X</td>
-      <td>No</td>
+      <td>Thumbstick X</td>
+      <td>If touchpad is not present</td>
     </tr>
     <tr>
       <td>axes[3]</td>
-      <td>Secondary Touchpad/Joystick Y</td>
-      <td>No</td>
+      <td>Thumbstick Y</td>
+      <td>If touchpad is not present</td>
     </tr>
   </tbody>
 </table>
 
 Devices that lack one of the optional inputs listed in the tables above MUST preserve their place in the {{Gamepad/buttons}} or {{Gamepad/axes}} array, reporting a <dfn>placeholder button</dfn> or <dfn>placeholder axis</dfn>, respectively. A [=placeholder button=] MUST report <code>0</code> for {{GamepadButton/value}}, <code>false</code> for {{GamepadButton/pressed}}, and <code>false</code> for {{GamepadButton/touched}}. A [=placeholder axis=] MUST report <code>0</code>. [=Placeholder buttons=] and [=placeholder axis|axes=] MUST be omitted if they are the last element in the array or all following elements are also [=placeholder buttons=] or [=placeholder axis|axes=].
 
-Additional buttons or axes may be exposed after these reserved indices, and SHOULD appear in order of decreasing importance. Related axes (such both axes of a joystick) SHOULD be grouped and, if applicable, SHOULD appear in X, Y, Z order. If a device has both a touchpad and a joystick the UA MAY designate whichever it chooses to be the primary axis-based input. Buttons reserved by the UA or platform MUST NOT be exposed.
+Additional buttons or axes may be exposed after these reserved indices, and SHOULD appear in order of decreasing importance. Related axes (such as both axes of a thumbstick) MUST be grouped and, if applicable, MUST appear in X, Y, Z order. Buttons reserved by the UA or platform MUST NOT be exposed.
```

Comment:
In order to introduce further reserved indices in the future, we'd need some way to reconcile the additional indices. (I suppose this is true before this change as well)

Perhaps apps need to be explicit that they want an `xr-standard` mapping for their `Gamepad` - that way, we can later introduce an `xr-standard-2` mapping or such that pushes out into button/axis 5, 6, etc.

Should the `gamepad` attribute perhaps be `getGamepad(mapping)`?

Reply:
I'm pretty strongly against making that large of a change to this chunk of the API this late. (Though I don't object to introducing a `xr-standard-2` at some point if needed.)

I do think we'll likely outgrow the current shape of the gamepad API at some point, but when that time comes I have a hunch that we'll be better served by introducing something along the lines of an action mapping API rather than continue to hack more robust concepts onto an API that obviously never expected to deal with them. (And hopefully at that point we have a few years of insight into how developers are using OpenXR to help inform our direction.)

Reply:
Sounds good.

Even if we later do need to support what I said in the gamepad API, there are slightly less clean ways to do it later without changes now - for example, when requesting your `XRSession`, you could centrally set `xr-standard-2` as your preferring mapping for receiving `xr-standard*` gamepad buttons/axes.